src/components/ColorPicker/ColorPicker.tsx

```diff
@@ -67,25 +110,44 @@ export class ColorPicker extends React.PureComponent<ColorPickerProps, State> {
       />
     ) : null;
 
+    const hexPickerMarkup = (
+      <TextPicker
+        color={color}
+        allowAlpha={allowAlpha}
```

Comment:
Instead of having to pass an `allowAlpha`, which the `TextPicker` shouldn't really care about imo. Could we render them in a grid or even a flex container and let this component take a bit more space than the alpha field?

Reply:
One issue with not passing it is the use of `allowAlpha` in the [TextFieldSwatch](https://github.com/Shopify/polaris-react/blob/8c6f48f5a1213bd18cb541c3841b9968af100130/src/components/ColorPicker/ColorPicker.scss#L187).